From 66c65d904da9d56b02bcfdc9c6827ef2d00bdf65 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@quansight.com>
Date: Fri, 1 Aug 2025 13:31:58 +0200
Subject: [PATCH 3/5] Use system PATH to find tools (in CONDA_PREFIX)

---
 python/triton/knobs.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/python/triton/knobs.py b/python/triton/knobs.py
index d3cbc9ac8f..6be5dd3655 100644
--- a/python/triton/knobs.py
+++ b/python/triton/knobs.py
@@ -4,6 +4,7 @@ import functools
 import importlib
 import os
 import re
+import shutil
 import subprocess
 import sysconfig
 import pathlib
@@ -207,9 +208,9 @@ class env_nvidia_tool(env_base[str, NvidiaTool]):
         # We still add default as fallback in case the pointed binary isn't
         # accessible.
         if path is not None:
-            paths = [path, self.default_path]
+            paths = [path, self.default_path, shutil.which(self.binary)]
         else:
-            paths = [self.default_path]
+            paths = [self.default_path, shutil.which(self.binary)]
 
         for path in paths:
             if tool := NvidiaTool.from_path(path):
